suite/tests/client-interface/drx-scattergather.c

```diff
@@ -153,6 +157,35 @@ test_avx2_vgatherqpd(uint32_t *ref_sparse_test_buf, uint32_t *test_idx32_vec,
 #    define CONCAT_XMM_YMM_U32 ((XMM_REG_SIZE + YMM_REG_SIZE) / sizeof(uint32_t))
 #    define SPARSE_TEST_BUF_SIZE_U32 (SPARSE_FACTOR * ZMM_REG_SIZE / sizeof(uint32_t))
 #    define POISON 0xf
+#    define CPUID_KMASK_COMP 5
+
+#    ifdef UNIX
+static SIGJMP_BUF mark;
+
+static int
+get_xstate_area_offs(int comp)
```

Comment:
Not sure what "comp" stands for: is there a more clear name which could be used?

Reply:
s/comp/xstate_component/